tests/bluetooth/bsim_bt/bsim_test_mesh/src/oob_client.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+
+#define LOG_MODULE_NAME udp_client
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#define PORT    26013
+
+static int sockfd;
+static struct sockaddr_in servaddr;
+
+int oob_client_init(void)
+{
+	/* Creating socket file descriptor */
+	sockfd = socket(AF_INET, SOCK_DGRAM, 0);
+	if (sockfd < 0) {
+		LOG_INF("Socket creation failed");
+		return sockfd;
+	}
+
+	LOG_INF("Socket with UDP client has been created");
+
+	return 0;
+}
+
+int oob_data_send(char *buffer, int len)
+{
+	/* Filling server information */
+	servaddr.sin_family = AF_INET;
+	servaddr.sin_port = htons(PORT);
+	servaddr.sin_addr.s_addr = INADDR_ANY;
+
+	LOG_INF("OOB UDP data tx");
+
+	int err = sendto(sockfd, buffer, len,
+		MSG_CONFIRM, (const struct sockaddr *)&servaddr, sizeof(servaddr));
+
+	close(sockfd);
+
+	return err;
+}
```

Comment:
@trond-snekvik There is already :): https://github.com/BabbleSim/base/blob/master/libPhyComv1/src/bs_pc_backchannel.c#L7

One thing you should be aware of, when you build this kind of solution (like in this PR), with just 1 of something (one socket, one file, one whathaveyou) is that you cannot run multiple simulations in parallel. Not with different sim_ids, and not by different users in the same machine, or sharing the same drive/folder. That can be a significant problem, for ex., if you want to run things in parallel for speed,  for CI systems, or so. The basic solution for this in the libraries provided with bsim is that results/dump/intermediate files are all kept in the results/<sim_id>/ folder. And any temporary "files" are kept in /tmp/bs_<user>/<sim_id>/. You don't need to use that if you don't want to. But if you don't, you should be mindful of not shooting your foot to badly.

Reply:
Really I do not want the universe spaceship here. I added the OOB interface according to Mesh spec usage. This is p2p communication to provide simple data and closes the socket immediately. Do not use it for BubbleSim debug purposes it will leave your foots intact :)

Reply:
@alxelax What happens if 2 users in the same computer run the same simulation at the same time, or 2 simulations with 2 different sim_ids happen to use this OOB mechanism at the same time? :)

Reply:
@aescolar That is quite obvious since UDP is the transport layer. All port listeners will receive the same data. The problem that you described is not unique and has been solved a long time ago by creating an extended ISO model. There are Session and Presentation layers to solve that. The question is how possible is the situation in real life for these tests that you described here? Are there real use cases?

Reply:
It seems to me like the existing backchannel solution would be the right approach here, even if the novel solution in this file works properly. 

@aescolar is there some sample or existing application that uses this, that we can use as a reference? Now that I know this feature exists, I'd like to use it actively in tests. I suppose it could even be integrated into the test framework with a standard channel for creating sync points.

Reply:
Ok. I'll change my implementation because this is a code owner request. However, I still think it is not a good idea. Trying to fix what is not broken basing on strange assumptions.

Reply:
@trond-snekvik 
> is there some sample or existing application that uses this, that we can use as a reference?

Unfortunately nothing open sourced. I would hope that the description/comments in that file (see each function comment) would be good enough to get started. The functionality the backchannels provide is quite simple.